addons/web/static/tests/legacy/components/action_menus_tests.js

```diff
@@ -246,5 +246,34 @@ odoo.define('web.action_menus_tests', function (require) {
 
             actionMenus.destroy();
         });
+
+        QUnit.test('execute action with context', async function (assert) {
+            assert.expect(3);
+
+            const actionMenus = await createComponent(ActionMenus, {
+                env: {
+                    action: this.action,
+                    view: this.view,
+                },
+                props: {
+                    isDomainSelected:  true,
+                    ...this.props
+                },
+                async mockRPC(route, args) {
+                    if (route === "/web/dataset/call_kw/hobbit/search"){
+                        assert.step('search');
+                        assert.ok("context" in args.kwargs, "kwargs must contains context");
```

Comment:
I suggest to have a more precise assertion here (context might actually always be the empty object, which is wrong, but the test would pass):
```js
assert.deepEqual(args.kwargs.context, { some_key: "44" });
```
Obviously, it requires to setup the ActionMenus with the adequate context.